importers.md

```diff
@@ -0,0 +1,44 @@
+# Importers
+
+<!-- vim-markdown-toc GFM -->
+
+* [YSO - General Finnish ontology **(Required)**](#yso---general-finnish-ontology-required)
+  * [What is it?](#what-is-it)
+  * [What depends on it?](#what-depends-on-it)
+  * [What does the importer do?](#what-does-the-importer-do)
+
+<!-- vim-markdown-toc -->
+
+## YSO - General Finnish ontology **(Required)**
+
+### What is it?
+First, some terminology is in order. Starting with *ontology*:
+
+> Ontology is the philosophical study of being. More broadly, it studies concepts that directly relate to being, in particular
+> becoming, existence, reality, as well as the basic categories of being and their relations.
+>
+> From https://en.wikipedia.org/wiki/Ontology
```

Comment:
While this is decidedly cool, you stilll might want to use the computer sciencey definition instead: https://en.wikipedia.org/wiki/Ontology_(information_science)

Reply:
Done in ab9735d3005db6493e2ff0fa2b3962594c392afa